Marsiple/Common/Models/Comment.swift

```diff
@@ -22,4 +22,29 @@ class Comment: Codable {
         self.body = body
         self.postId = postId
     }
+    
+    private enum CodingKeys: String, CodingKey {
```

Comment:
We don't need the `CodingKeys` specified explicitly. `iOS` can infer them automatically.

Reply:
Is there another way to instantiate `container` than using CodingKeys like this: `var container = encoder.container(keyedBy: CodingKeys.self)`? I haven't found another one so I was forced to use `CodingKeys`.

Reply:
That's fine, but you get `CodingKeys.self` implicitly, when you conform the type to `Codable` (if all of the conforming type properties are `Codable` too). 

That's why you don't have to define the `CodingKeys` enum explicitly, you get it for "free" 🙂